pom.xml

```diff
@@ -637,7 +637,7 @@
             <dependency>
                 <groupId>io.airlift.tpch</groupId>
                 <artifactId>tpch</artifactId>
-                <version>0.9</version>
+                <version>0.10</version>
```

Comment:
I saw you updated `tpch` but it's not obvious how airbase update here relates to tpch version bump.

Reply:
I'll update the description. The old version was not compatible with the new version of Guava.